### DISCUSSION TITLE
Account suspended

### DISCUSSION THREAD

**Discussion by DrKingsuk** [community/community#110366] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Here a message show that your account has been suspended to violate term and condition  when I login in my account. I was email GitHub team but they refer to me this community. They didn't send any solution pleased help me

**Comment by queenofcorgis** [comment, marked as answer]
Hi @DrKingsuk,

The community is unable to assist with nor escalate your situation, the best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :). It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
